FastSimulation/Calorimetry/src/CalorimetryManager.cc

```diff
@@ -180,71 +181,82 @@ void CalorimetryManager::reconstruct(RandomEngineAndDistribution const* random)
       if(debug_)
 	mySimEvent->print();
     }
+
+  initialize();
+  
+  LogInfo("FastCalorimetry") << "Reconstructing " << (int) mySimEvent->nTracks() << " tracks." << std::endl;
```

Comment:
Is this LogInfo linked to the ones below L194-195? If so, shouldn't they be either all kept uncommented, or commented out (better: removed?)

Reply:
All the commented lines, couts etc already exited in the code - all I did was to add a new method/constructur to the class. I will go through the code and clean it from all unnecessary lines.